docs/xls-75d-delegating-account-permissions/index.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Please update this to use the feature landing page template, for example https://github.com/ripple/opensource.ripple.com/blob/main/docs/xls-65d-single-asset-vault/index.page.tsx